src/services/item/service.ts

```diff
@@ -95,7 +96,7 @@ export class ItemService {
     member: Member,
     repositories: Repositories,
     args: {
-      item: Partial<Item> & Pick<Item, 'name' | 'type'>;
+      item: Partial<Omit<Item, 'geolocation'>> & Pick<Item, 'name' | 'type'>;
```

Comment:
Maybe we can set `geolocation` to `null` or `undefined`, otherwise I don't understand why you we shouldn't omit `creator` for example.